db/schema.yml

```diff
@@ -681,14 +681,14 @@ container_groups:
 - dns_policy
 - ems_id
 - container_node_id
-- last_perf_capture_on
 - container_replicator_id
 - ipaddress
 - type
 - container_project_id
 - phase
 - message
 - reason
+- last_perf_capture_on
```

Comment:
👎  Column order should not change, so this is wrong (and will break replication).  cc @carbonin @lpichler